package-lock.json

```diff
@@ -6631,8 +6631,9 @@
     },
     "node_modules/restructure": {
       "version": "3.0.0",
-      "resolved": "https://registry.npmjs.org/restructure/-/restructure-3.0.0.tgz",
-      "integrity": "sha512-Xj8/MEIhhfj9X2rmD9iJ4Gga9EFqVlpMj3vfLnV2r/Mh5jRMryNV+6lWh9GdJtDBcBSPIqzRdfBQ3wDtNFv/uw=="
+      "resolved": "git+ssh://git@github.com/keymanapp/dependency-restructure.git#49d129cf0916d082a7278bb09296fb89cecfcc50",
```

Comment:
Yes, this is a generated file… but `git+ssh` is unexpected here for a public repo.  Seems like this could cause headaches in CI etc downstream.  Is there something local that's causing this on your end?

Reply:
NPM has done this automatically. Not sure we can do much about it.